ThirdPartyAdapters/vungle/vungle/src/main/java/com/vungle/mediation/VungleManager.java

```diff
@@ -120,11 +120,9 @@ boolean isValidPlacement(String placementId) {
     }
 
     VungleNativeAd getVungleNativeAd(String placement, AdConfig adConfig, VungleListener vungleListener) {
-        Log.d(TAG, "getVungleNativeAd");
-
         cleanUpBanner(placement);
         //Fetch new ad
-
+        Log.d(TAG, "getVungleNativeAd");
```

Comment:
move this log here just make logs seems easy to understand: getVungleNativeAd called after cleanUpBanner